cumulus/parachains/integration-tests/emulated/common/src/macros.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[macro_export]
+macro_rules! test_parachain_is_trusted_teleporter {
+	( $sender_para:ty, vec![$( $receiver_para:ty ),+], ($assets:expr, $amount:expr) ) => {
+		$crate::paste::paste! {
+			// init Origin variables
+			let sender = [<$sender_para Sender>]::get();
+			let mut para_sender_balance_before =
+				<$sender_para as $crate::Chain>::account_data_of(sender.clone()).free;
+			let origin = <$sender_para as $crate::Chain>::RuntimeOrigin::signed(sender.clone());
+			let fee_asset_item = 0;
+			let weight_limit = $crate::WeightLimit::Unlimited;
+
+			$(
+				{
+					// init Destination variables
+					let receiver = [<$receiver_para Receiver>]::get();
+					let para_receiver_balance_before =
+						<$receiver_para as $crate::Chain>::account_data_of(receiver.clone()).free;
+					let para_destination =
+						<$sender_para>::sibling_location_of(<$receiver_para>::para_id()).into();
+					let beneficiary =
+						$crate::AccountId32 { network: None, id: receiver.clone().into() }.into();
+
+					// Send XCM message from Origin Parachain
```

Comment:
This is just calling `limited_teleport_assets`, which is fine, but the comment should say that.

Reply:
Added comment